### PR TITLE
Started writing up v3 README

### PR DESCRIPTION
**Pull request by scott-w**
### Proposed changes
 - Changes to the README for v3

### Summary

I'm using this PR to propose changes to the README. If there's anything you think is important, put it here and I'll update this PR with the information.

### CODE REVIEW

**Comment by JSteunou** [comment]
So exciting :fireworks:

File: readme.md [hunk 1]
```diff
@@ -126,24 +138,23 @@ You can read the annotations for all the details of how Marionette works, and ad
 MarionetteJS currently works with the following libraries:
 
 * [jQuery](http://jquery.com) v1.8+
-* [Underscore](http://underscorejs.org) v1.4.4 - 1.6.0
-* [Backbone](http://backbonejs.org) v1.0.0 - 1.1.2 are preferred.
-* [Backbone.Radio](https://github.com/marionettejs/backbone.radio) Comes automatically with the bundled build.
-* [Backbone.BabySitter](https://github.com/marionettejs/backbone.babysitter) Comes automatically with the bundled build.
+* [Underscore](http://underscorejs.org) v1.8.3
```
Comment (rafde) [inline readme.md]: Lodash 4 core compatible?

File: readme.md [hunk 2]
```diff
@@ -80,30 +84,38 @@ We're happy to discuss design patterns and learn how you're using Marionette.
 
 <a href="https://github.com/marionettejs/marionette.inspector"><img src="http://i.imgur.com/B1q9QXH.jpg" align="center" /></a>
```
Comment (paulfalgout) [inline readme.md]: so the inspector is kind of broken right now and isn't going to work with v3.. maybe we should pull it for now?

File: readme.md [hunk 3]
```diff
@@ -126,24 +138,23 @@ You can read the annotations for all the details of how Marionette works, and ad
...
 
 * [jQuery](http://jquery.com) v1.8+
-* [Underscore](http://underscorejs.org) v1.4.4 - 1.6.0
-* [Backbone](http://backbonejs.org) v1.0.0 - 1.1.2 are preferred.
-* [Backbone.Radio](https://github.com/marionettejs/backbone.radio) Comes automatically with the bundled build.
-* [Backbone.BabySitter](https://github.com/marionettejs/backbone.babysitter) Comes automatically with the bundled build.
+* [Underscore](http://underscorejs.org) v1.8.3
+* [Backbone](http://backbonejs.org) v1.3.3
+* [Backbone.Radio](https://github.com/marionettejs/backbone.radio) Comes bundled (v1.0.0)
+* [Backbone.BabySitter](https://github.com/marionettejs/backbone.babysitter) Comes bundled (v1.0.0)
```
Comment (paulfalgout) [inline readme.md]: this line can go :-)

Comment (scott-w) [inline readme.md]: Shall I chop the Radio line too?

Comment (paulfalgout) [inline readme.md]: I didn't actually remove Radio as a dependency in package.json.  I still kinda think it's cool to keep it as default.  People can alias it out if need be right?  But `<script>` tag users can add Mn without having to worry about Radio day one.

**Pull request merged**